test/deprecates.jl

```diff
@@ -0,0 +1,201 @@
+using StatsBase
+using Base.Test
+using Compat
+import Compat: view
+
+@testset "StatsBase.Deprecates" begin
+
+@testset "Deprecates WeightVec and weights" begin
```

Comment:
I'd rather not. Deprecated code is supposed to be removed relatively soon anyway.